autoload/ctrlspace/help.vim

```diff
@@ -312,8 +312,23 @@ function! s:keyHelp(key, description)
     endif
 endfunction
 
+function! s:sortKeyHelp(key1, key2) dict
+    if has_key(s:descriptions, self[a:key1])
+        if has_key(s:descriptions, self[a:key2])
+            let l:desc1 = s:descriptions[self[a:key1]]
+            let l:desc2 = s:descriptions[self[a:key2]]
+            return l:desc1 == l:desc2 ? 0 : l:desc1 > l:desc2 ? 1 : -1
+        else
+            return -1
+        endif
```

Comment:
This can be destructured into an unnested-if I believe, where you just check for `has_key(s:descriptions, self[a:key1]) && has_key(s:descriptions, self[a:key2])`.

Reply:
Not quite, but I changed the structure to use guards, which is a better practice.